satzung.rst

```diff
@@ -135,7 +135,7 @@ Die Vertrauenspersonen dienen als Anlaufstelle für hilfesuchende Personen, die
 Ausgrenzung, Diskriminierung oder Belästigung im Rahmen der ZaPF erfahren haben.
 
 Die Wahl der höchstens sechs Vertrauenspersonen ist zu Beginn jeder ZaPF durchzuführen.
-Ihre Amtszeit endet mit dem Ende des Abschlussplenums oder Niederlegung des Amtes.
+Ihre Amtszeit endet mit der Neuwahl der Vertrauenspersonen oder Niederlegung des Amtes.
```

Comment:
Wenn die Vertrauenspersonen länger als eine ZaPF im Amt bleiben sollen, sollte für sie auch die gleiche Klausel bezüglich dem Ende der Amtszeit gelten, die auch für alle anderen Ämter in Organen der ZaPF gilt. Konkret bedeutet das, dass auch hier ein Ableben der amtsinnehabenden Person oder eine Abwahl durch das Plenum die Amtszeit beendet.

Wenn es gewünscht ist, kann ich gleich mal einen Vorschlag bauen, der das widerspiegeln würde

Reply:
Hab jetzt das Ableben der Amtsinhaber*innen noch dazu genommen. Abwahl macht in dem Fall keine Sinn, weil im nächsten Plenum, in dem sie abgewählt werden können, sind sie eh nicht mehr Vertrauensperson.

Reply:
Ich halte es schon für eine sinnvolle Regelung, da Vertrauenspersonen auch auf dem Zwischen- oder Abschlussplenum abgewählt werden könnten.

Außerdem blieben bei eurer aktuellen Formulierung die alten Vertrauenspersonen im Amt wenn keine neuen auf der nächsten ZaPF gewählt werden. Angesichts dessen halte ich schon für nötig, Vertrauenspersonen abwählen zu können.